.travis.yml

```diff
@@ -24,10 +24,7 @@ notifications:
 
 before_install:
   - gem update bundler
-  - .travis/install_gradle.sh
-  - .travis/install_rebar.sh
-  - .travis/install_bower.sh
-  - .travis/install_godep.sh
+  - for s in .travis/install_*.sh; do $s; done
```

Comment:
Well, if ordering *would* matter, that would not be obvious from the original code either. IMO that would need to be documented, e.g. by adding padded number prefixes to the scripts, like `install_0001_gradle.sh` and so on. And if that was the case, my new code would run the scripts in order, too, thanks to alphabetical ordering :-)